jetstream/statistics.py

```diff
@@ -908,7 +913,7 @@ def transform(
         return StatisticResultCollection.parse_obj(results)
 
 
-@attr.s(auto_attribs=True)
+@attr.s(auto_attribs=True, kw_only=True)
```

Comment:
Why's this needed here? And why not for the other stats?

Reply:
This Stat needs it because it has positional arguments without defaults (`numerator` and `denominator`), which is not allowed in a subclass where the parent class has arguments with defaults. This [stackoverflow](https://stackoverflow.com/questions/72191842/python-attrs-inheriting-from-class-which-values-have-default-values-raises-erro) question goes into more detail. 

Since we [only ever ](https://github.com/mozilla/jetstream/blob/main/jetstream/statistics.py#L303-L305) construct these objects using keywords anyway, I figured this would be alright

Reply:
Ah yep that makes sense, thanks!